static/style.css

```diff
@@ -95,3 +95,20 @@ pre {
   border: 3px solid #a4a0e8;
   border-radius: 3px;
 }
+
+.selling-points {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.selling-point {
+  /* 3 selling points per row */
+  width: 26%;
```

Comment:
```suggestion
  /* 2 selling points per row */
  max-width: 20em;
```
I think having two selling points will make it look better and setting the max width.

![image](https://user-images.githubusercontent.com/4687791/158040977-29550baf-02a0-4814-a1ec-05c93cd0043a.png)

![image](https://user-images.githubusercontent.com/4687791/158040983-fb2d23e6-d1e2-46e2-8037-7fff495e6f2a.png)

Reply:
Thanks for the suggestion, looks much better 👍🏾